src/ImageRunModal.jsx

```diff
@@ -635,6 +638,75 @@
         return owner === systemOwner;
     };
 
+    prefillModal() {
+        const container = this.props.container;
+        const containerDetail = this.props.containerDetail;
+        const image = this.props.localImages.find(img => img.Id === container.ImageID);
+        const owner = container.isSystem ? 'system' : this.props.user;
+
+        if (containerDetail.Config.CreateCommand) {
+            this.setState({
+                dialogWarning: _("This container was not created by cockpit"),
+                dialogWarningDetail: _("Some options may not be copied to the new container."),
+            });
+        }
+
+        const env = containerDetail.Config.Env.filter(variable => {
+            if (image.Env.includes(variable)) {
+                return false;
+            }
+
+            return !variable.match(/((HOME|TERM|HOSTNAME)=.*)|container=podman/);
+        }).map((variable, index) => {
+            const split = variable.split('=');
+            return { key: index, envKey: split[0], envValue: split[1] };
+        });
+
+        const publish = container.Ports
+            ? container.Ports.map((port, index) => {
+                return { key: index, IP: port.hostIP || port.host_ip, containerPort: port.containerPort || port.container_port, hostPort: port.hostPort || port.host_port, protocol: port.protocol };
+            })
+            : [];
+
+        const volumes = containerDetail.Mounts.map((mount, index) => {
+            // podman does not expose SELinux labels
+            return { key: index, containerPath: mount.Destination, hostPath: mount.Source, mode: (mount.RW ? 'rw' : 'ro'), selinux: '' };
+        });
+
+        // check if memory and cpu limitations or healthcheck are used
+        const memoryConfigure = containerDetail.HostConfig.Memory > 0;
+        const cpuSharesConfigure = containerDetail.HostConfig.CpuShares > 0;
+        const healthcheck = !!containerDetail.Config.Healthcheck;
+        const healthCheckOnFailureAction = (this.props.version.split(".")) >= [4, 3, 0]
+            ? HealthCheckOnFailureActionOrder.find(item => item.apiName === containerDetail.Config.HealthcheckOnFailureAction).value
+            : null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230615-141749-dd990807-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)